http/middleware_test.go

```diff
@@ -0,0 +1,67 @@
+package http
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/clerk/clerk-sdk-go/v2"
+	"github.com/stretchr/testify/require"
+)
+
+func TestWithHeaderAuthentication_InvalidAuthorization(t *testing.T) {
```

Comment:
🙃  Looks like there's a mix-up between Authentication and Authorization, even in the PR description. But Authorization seems to be the right term for this case.

```suggestion
func TestWithHeaderAuthorization_InvalidAuthorization(t *testing.T) {
```

Reply:
Updated!